openacc/nemolite2d.f90

```diff
@@ -420,6 +422,7 @@ SUBROUTINE continuity
                 ssha(ji, jj) = sshn(ji, jj) + (rtmp2 - rtmp1 + rtmp4 - rtmp3)*rdt/e12t(ji, jj)
             END DO
         END DO
+        !$acc end parallel
```

Comment:
Where's the end loop? Also if you're doing both at the same time you can just do `!$acc parallel loop` followed by `!$acc end parallel loop`

Reply:
If you're doing parallel and loop the same you can just do `!$acc parallel loop` instead of using two separate directives, but I don't think it makes any difference.

Reply:
fixed!